### ISSUE TITLE
Constrained Baseline profile support

### ISSUE THREAD

**Issue by pascalbaljet** [pedroSG94/rtmp-rtsp-stream-client-java#92]
Is there any way I can change the default `Baseline` profile to the `Constrained Baseline` profile?

**Comment by pedroSG94** [comment]
I don't understand the question. 
What is baseline and constrained baseline?

**Comment by pascalbaljet** [comment]
By default the H264 profile of the `SrsFlvMuxer` is set to the Baseline Profile:
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/blob/master/rtmp/src/main/java/net/ossrs/rtmp/SrsFlvMuxer.java#L445

The Constrained Baseline Profile is a subset of the Baseline Profile, it also has a value of 66. I need this Constrained Baseline Profile because I'm using the stream for a low-latency setup. 

See also: https://en.wikipedia.org/wiki/H.264/MPEG-4_AVC#Profiles

**Comment by pedroSG94** [comment]
Ah, baseline of h264.
But constrained baseline and baseline has same value. 66(0x42)

**Comment by pascalbaljet** [comment]
Alright I’ll dig deeper into the project and try to come up with a solution. Thanks for the hard work, great project!

**Comment by pedroSG94** [comment]
videoFormat.setInteger("profile",MediaCodecInfo.CodecProfileLevel.AVCProfileConstrainedBaseline);
videoFormat.setInteger("profile",65536);

try with it you need api 27 but you can try hardcode value. Write it on VideoEncoder class

https://stackoverflow.com/questions/22611451/set-avc-h-264-profile-when-encoding-video-in-android-using-mediacodec-api

**Comment by pascalbaljet** [comment]
After some debugging I found the problem! The `profile-level-id` had the wrong `profile_iop` parameter. It had a constant value but I submitted a pull request to make it variable: https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/pull/93

Apparently the right value for the Constrained Baseline profile is 192. This pull request allows you to do this:

```java
RtmpCamera2 rtmpCamera2 = new RtmpCamera2(mTextureView, new ConnectCheckerRtmp());

rtmpCamera2.setProfileIop((byte) 0xC0);
```
